pkg/agent/flowexporter/utils.go

```diff
@@ -91,3 +103,14 @@ func PolicyTypeToUint8(policyType v1beta2.NetworkPolicyType) uint8 {
 		return registry.PolicyTypeK8sNetworkPolicy
 	}
 }
+
+// LookupProtocolMap returns protocol identifier given protocol name
+func LookupProtocolMap(name string) (uint8, error) {
```

Comment:
not introduced by your PR, but really this function belongs in `pkg/util/ip/ip.go`. We already have the reverse function (`IPProtocolNumberToString`) there. That being said, it can be postponed to a future PR.

Reply:
we can move it in the future PR